wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java

```diff
@@ -353,6 +353,7 @@ private void renderCombinedEventScripts()
 		}
 		if (combinedScript.length() > 0)
 		{
+			combinedScript.append("\nWicket.Event.publish(Wicket.Event.Topic.AJAX_TIMERS_BOUND);");
```

Comment:
Yes, I think we should publish both for 8.x and remove `AJAX_HANDLERS_BOUND` for 9.x.
We can also log a warning when `AJAX_HANDLERS_BOUND` published to a subscriber.